src/core/index.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
     return config;
   },
   getAvailableTemplates: function () {
-    return ["nodeJs"];
+    return ["nodeJs", ".env"];
```

Comment:
The `.env` config here specifies adding a .env file or adds the env file loading support to the exisiting node project? Could we make the name a bit more explicit to denote that?